data/map_intializer.py

```diff
@@ -7,16 +7,17 @@
 class MapInitializer:
     def __init__(self, path_to_img: str, image_loader: ImageLoader = ImageLoader()):
         self.image = image_loader._get_image(path_to_img)
+        self.image = self.image.transpose(1,0,2)
 
     def get_image_size(self):
         return np.shape(self.image)[:2]
 
     def get_cell_array(self, cell_size: int):
         (X, Y, Z) = np.shape(self.image)
         array = np.empty(shape=(X, Y), dtype=np.dtype(object))
-        for y in range(Y):
-            for x in range(X):
-                pixel_rgb = self.image[y,x]
+        for x in range(X):
+            for y in range(Y):
```

Comment:
Bardziej pythonowo by było zrobić:
`for x, row in enumerate(self.image):
            for y, pixel in enumerate(row:)`

Potem nie musisz robić pixel_rgb = self.image[x,y]